
def expect_exception(e, c):
    try:
        c()
        return False
    except e:
        return True


